### ISSUE TITLE
Create Category

### ISSUE THREAD

**Issue by soulgin** [soulgin/Calendar-task-planner-app#23]
Create Category: <!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Create Category</title>
    
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" />
  </head>
  <body>
    <div class="container mt-5">
      <h2  class="text-center">Create Category</h2>
      <br>
      <form method="post" action="{% url 'create_category' %}">
        {% csrf_token %}
        <div class="form-group">
          <label for="categoryName">Category Name:</label>
          <input type="text" class="form-control" id="categoryName" name="name" placeholder="Enter category name" required />
        </div>
        <button type="submit" class="btn btn-primary">Create</button>
      </form>
    </div>

    
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
  </body>
</html>
